diff --git a/core/modules/field/src/Plugin/migrate/process/d7/FieldInstanceSettings.php b/core/modules/field/src/Plugin/migrate/process/d7/FieldInstanceSettings.php
index e400991..a4334cb 100644
--- a/core/modules/field/src/Plugin/migrate/process/d7/FieldInstanceSettings.php
+++ b/core/modules/field/src/Plugin/migrate/process/d7/FieldInstanceSettings.php
@@ -48,7 +48,7 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
         'field' => '_none',
         'direction' => 'ASC',
       ];
-      if (!empty(array_filter($field_settings['handler_settings']['sort']))) {
+      if (!empty(array_filter($field_settings['handler_settings']['sort'] ?? []))) {
         if ($field_settings['handler_settings']['sort']['type'] == "property") {
           $sort = [
             'field' => $field_settings['handler_settings']['sort']['property'],
